models/biocro/R/model2netcdf.BIOCRO.R

```diff
@@ -71,14 +72,19 @@ model2netcdf.BIOCRO <- function(result, genus = NULL, outdir, lat = -9999, lon =
     
     k <- udunits2::ud.convert(1, "Mg/ha", "kg/m2") / c2biomass
     
-    RR <- with(R, list(TotLivBiom = k * (Leaf + Root + Stem + Rhizome + Grain), 
-                       RootBiom = k * Root, 
-                       StemBiom = k * Stem, 
-                       Yield = Stem,
-                       Evap = udunits2::ud.convert(SoilEvaporation + CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
-                       TVeg = udunits2::ud.convert(CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
-                       LAI = LAI))
+    result_yeari_std <- with(result_yeari, list(
+      TotLivBiom = k * (Leaf + Root + Stem + Rhizome + Grain), 
+      RootBiom = k * Root, 
+      StemBiom = k * Stem, 
+      Evap = udunits2::ud.convert(SoilEvaporation + CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
+      TVeg = udunits2::ud.convert(CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
+      LAI = LAI))
     
+    total_biomass <- with(result_yeari, 
+                          k * (TotLivBiom + Leaf + Root + Stem + Rhizome + Grain + AboveLitter + BelowLitter))
```

Comment:
1) If TotLivBiom is Leaf + Root + Stem + Rhizome + Grain, doesn't that cause these terms to be double counted?
2) It's still unclear to me whether TotLivBiom is already a column in results (in which case, why redefine it above in result_yeari_std) or whether this calculation should really be done on result_yeari_std not result_yeari?